ci/ReleaseTool/ReleaseCommand.cs

```diff
@@ -213,16 +213,6 @@ public int Run()
                     gitClient.Fetch();
                     gitClient.CheckoutRemoteBranch(options.ReleaseBranch);
                     FinalizeRelease(gitHubClient, gitClient, gitHubRepo, gitRepoName, repoUrl);
-                    var release = CreateRelease(gitHubClient, gitHubRepo, gitClient, repoName);
```

Comment:
Don't immediately see how this fixes the issue?

Reply:
FinalizeRelease does exactly what this code block did. I guess I forgot to remove it when refactoring